frontend/static/js/process-editor.js

```diff
@@ -387,34 +408,49 @@ function getMetricRowTarget(innerHTML_target, actual_target_metrics, slug) {
     });
 
     // replace null with empty strings, so that "null" is not entered in the table
-
-    innerHTML_target['min'] = `
+    if (!binary) {
+        innerHTML_target['min'] = `
                         <td class="col-7" ><input type="text" name="target-minimum" id = "` + slug + `" value="` + targetValues['min'] + `"`;
-    innerHTML_target['max'] = `
+        innerHTML_target['max'] = `
                         <td class="col-8" ><input type="text" name="target-maximum" id = "` + slug + `" value="` + targetValues['max'] + `"`;
-    innerHTML_target['average'] = `
-                        <td class="col-9" ><input type="text" name="target-average" id = "` + slug + `" value="` + targetValues['average'] + `"`;
+        innerHTML_target['average'] = `
+                        <td class="col-9" ><input type="text" name="target-average" + id = "` + slug + `" value="` + targetValues['average'] * 100 + `"`;
+    } else {
+        innerHTML_target['min'] = `
+                        <td class="col-7" ><input type="text" name="target-minimum" id = "` + slug + `" disabled`;
+        innerHTML_target['max'] = `
+                        <td class="col-8" ><input type="text" name="target-maximum" id = "` + slug + `" disabled`;
+        innerHTML_target['average'] = `
+                        <td class="col-9" ><input type="text" name="target-average" binary="true" + id = "` + slug + `" value="` + targetValues['average'] * 100 + `"`;
+    }
```

Comment:
Hier bitte nochmal drüber schauen, wie bei der Funktion darüber auch den HTML Code nur einmal in eine Variable packen und nicht doppelt stehen haben. Bei Fragen gerne nochmal schreiben :)
Sonst schaut es eigentlich gut aus :)

Reply:
An der Stelle hielten Nicolas und ich das nicht für so sinnvoll, weil ja auch das disabled-Attribut dann über eine Variable gesetzt werden muss und es dann wahrscheinlich eher unübersichtlicher wird. Wenn das doch nötig ist, können wir das morgen noch machen :)

Reply:
Okay dann passt